src/Sulu/Component/Webspace/Document/Initializer/WebspaceInitializer.php

```diff
@@ -64,9 +64,20 @@ public function __construct(
         $this->nodeManager = $nodeManager;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function initialize(OutputInterface $output)
     {
-        foreach ($this->webspaceManager->getWebspaceCollection() as $webspace) {
+        // sort the webspaces in order that they always appear
+        // in the same order across different platforms (which
+        // helps when testing).
+        $webspaces = iterator_to_array($this->webspaceManager->getWebspaceCollection());
```

Comment:
I know, I just wanted to see which tests fail.

On Thu, Apr 14, 2016 at 12:14:01AM -0700, Daniel Rotter wrote:
>    In
>    [1]src/Sulu/Component/Webspace/Document/Initializer/WebspaceInitializer.php:
> 
>  >      public function initialize(OutputInterface $output)
>  >      {
>  > -        foreach ($this->webspaceManager->getWebspaceCollection() as $webspace) {
>  > +        // sort the webspaces in order that they always appear
>  > +        // in the same order across different platforms (which
>  > +        // helps when testing).
>  > +        $webspaces = iterator_to_array($this->webspaceManager->getWebspaceCollection());
> 
>    [2]@dantleech If you remove the usort you should also be able to remove
>    the iterator_to_array call.
> 
>    —
>    You are receiving this because you were mentioned.
>    Reply to this email directly or [3]view it on GitHub
> 
>    Reverse link: [4]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu/pull/2278#discussion_r59671582
>    2. https://github.com/dantleech
>    3. https://github.com/sulu/sulu/pull/2278/files/505dc324c989deb11f0bb95b4b04f70fbeb26b3d#r59671582
>    4. https://github.com/sulu/sulu/pull/2278/files/505dc324c989deb11f0bb95b4b04f70fbeb26b3d#r59671582